Readme.md

```diff
@@ -0,0 +1,48 @@
+     b. __``okta:ClientId``__: the ``Client ID`` value of your Okta OIDC Native app.      
+     c. __``okta:RedirectUri``__: a valid redirect uri as set up in your Okta OIDC Native app. This value should be of the form ``http://127.0.0.1:{any_port}/`` and configured as a redirect uri in your Okta OIDC app. __Important note: Make sure to include the trailing slash!__
+     d. __``okta:Scopes``__: the OpendID Connect scopes your application will request from Okta - you can use the default scopes as already configured.    
+     e. __``okta:ResponseType``__: the OpenID Connect response type (can currently be ``cpde`` or ``code id_token``)
+4. You can test the application with the parameters above and verify that you can sign in with Okta in your browser (or leverage an existing Okta session). You should be able to verify that your console application is able to authenticate you with the same credentials you used in the browser.
+5. If you want to test the ability to call an external API (for instance, our [ASP.NET Core Web API sample](https://github.com/raphaellondner-okta/okta-oauth-dotnetcore-rs-simple)), you must have access to Okta's API Access Management product, which is currently in beta version. If you want access to this product, please submit a request on the [Okta beta site](https://oktabeta.zendesk.com) and select ``API Access Management`` in the _Beta Name_ dropdown list.
+6. Once you've been granted access to Okta's API Access Management product, navigate to __Security-->API__ in the Admin dashboard of your Okta organization. You should see a page similar to the screenshot below:
+     ![Authorization Servers Home Page](https://github.com/raphaellondner-okta/okta-oauth-nativewindows/blob/google-console/assets/Okta_AuthorizationServersHomePage.PNG)
+7. Press the ``Add Authorization Server`` button and  a descriptive name (such as _ToDo List API_), a resource Uri (such as _http://todolist.example.com_), as well as an optional description. You should see aenter page similar to the screenshot below:
+        ![Todo List API Authorization Server](https://github.com/raphaellondner-okta/okta-oauth-nativewindows/tree/google-console/assets/Okta_AuthorizationServersHomePage.PNG)
+8. Take note of the __Issuer__ value on this page and copy/paste it to ``okta:AuthorizationServerUrl`` parameters in the __App.config__ file of this project.
+9. In Okta's Admin dashboard, select the _Scopes_ tab and select the ``Add Scope`` button. 
+     a. In the window that opens, enter ``todolist.read`` in the __Name__ field and ``Permission to read the Todo List`` in the __Description__ field    
+     You should end up with the following _Scopes_ tab:
+     ![Scopes tab](https://github.com/raphaellondner-okta/okta-oauth-nativewindows/blob/google-console/assets/Okta_TodoList.ReadScope.PNG)
+     b. In the __App.config__ file, update the __``okta:Scopes``__ value and append ``todolist.read`` to that list
```

Comment:
Again, incorrectly formatted. I'm going to stop calling these out - you should fix them above and in the rest of the document.